sklearn/neighbors/lof.py

```diff
@@ -148,7 +148,7 @@ def fit_predict(self, X, y=None):
         Returns
         -------
         is_inlier : array, shape (n_samples,)
-            Returns 1 for inliers and -1 for anomalies/outliers.
+            Returns 1 for anomalies/outliers and -1 for inliers.
```

Comment:
Is it related to dbscan ? If it is not, can you open another PR. 
BTW, it seems that this confusion is done in a lot of places in the LOF file (ex : [l138](https://github.com/scikit-learn/scikit-learn/pull/8039/files/176979e3f794eeece81a1ae60e1925a97b2e26ec..1b8c8c259f4006bf60cfd961529da60fb14b717c#diff-4985428b0bff6aa71f59fed1f8a72f93R138) ). 
Can you check the entire file ?